features/post/post-comment/post-comment.module.scss

```diff
@@ -0,0 +1,81 @@
+.comment {
+  display: flex;
+  gap: 5px;
+}
+
+.body {
+  display: flex;
+  flex-direction: column;
+  
+  &__text {
+    max-width: 345px;
```

Comment:
There is no need to manually limit the width of the comment. This should be done by the containing code through the container in which the component will be placed.